### DISCUSSION TITLE
Как работает режим прокси?

### DISCUSSION THREAD

**Discussion by maximusend** [dovecoteescapee/ByeDPIAndroid#99] [Q&A] [answered]
Привет!
Что из себя представляет режим "Proxy"? Трафик остальных приложений будет заворачиваться через ваше приложение автоматически или в каждом приложении нужно прописывать адрес прокси сервера?

**Comment by dovecoteescapee** [comment, marked as answer]
Режим прокси просто запускает [ByeDPI](https://github.com/hufrea/byedpi), как если бы вы его запустили на компьютере.
1. Его можно указать в настройках приложений, поддерживающих SOCKS-прокси. Если это приложение работает в режиме VPN (например, AdGuard) не забудь добавить ByeDPI в исключения, иначе трафик зациклиться и интернет не будет работать.
2. К нему можно подключиться с другого устройства по ip вашего телефона (его можно посмотреть в настройках телефона). В этом случае смените `Listen address` со `127.0.0.1` на `0.0.0.0` (или конкретный ip).

Обратите внимание, что в настройках сети на Android поддерживается только HTTP-прокси, туда указывать ByeDPI не нужно.
